src/org/oidc/common/WebFingerError.java

```diff
@@ -0,0 +1,7 @@
+package org.oidc.common;
+
+public class WebFingerError extends Exception {
```

Comment:
Naming custom exception as Error is misleading. Could you please make sure that name for custom exception end with Exception.

Instead of WebFingerError --> WebFingerException